app-forensics/volatility3/volatility3-2.11.0.ebuild

```diff
@@ -21,7 +21,7 @@ SRC_URI="
 "
 S="${WORKDIR}"/${PN}-${MY_PV}
 
-LICENSE="GPL-2+"
```

Comment:
git mv to revbump pls for the fixed licence as it changes the "freeness" of it (I wouldn't bother in some cases where the result is the same).